gpii/node_modules/flowManager/src/FlowManagerUtilities.js

```diff
@@ -66,6 +66,18 @@
             fluid.log("Fetched filtered solutions registry entries: " + JSON.stringify(solutionsRegistryEntries, null, 4));
             event.fire(solutionsRegistryEntries);
         });
+        if (deviceContext.OS.id !== "web") {
```

Comment:
Terrible! This is what we had even BEFORE we had object-orientation :) We had hard-wired if statements. Factor off all of this workflow into another grade - and try to figure out what grade it should be, given that we expect to have the hybrid (aka untrusted local flowManager) soon